label_studio/projects/models.py

```diff
@@ -487,14 +487,6 @@ def validate_config(self, config_string, strict=False):
         if not fields_from_config:
             logger.debug(f'Data fields not found in labeling config')
             return
-        fields_from_config = {field.split('[')[0] for field in fields_from_config}  # Repeater tag support
```

Comment:
can we comment with TODO: DEV-XXX reference?

Reply:
Added